config/config.defaults.yml

```diff
@@ -301,13 +301,37 @@ default: &default_settings
   # development and false for other environments.
   use_js_routes_middleware: false
 
-  # Custom landing page asset host, e.g. https://s3.amazonaws.com/my-bucket/sites
-  clp_asset_host:
+  # Path or URL for landing page assets.
+  #
+  # Because a directory is expected, it's recommended to include trailing slash.
+  #
+  # You can add %{sitename} in the URL and it will be replaced with the sitename,
+  # that is saved in the landing page structure JSON.
+  #
+  # Example values:
+  #
+  # - landing_page/
+  # - https://s3.amazonaws.com/my-bucket/sites/${sitename}/
```

Comment:
Example above is with `%{sitename}`, and here it's `${sitename}`. Which is the correct one?